crontab

```diff
@@ -1,2 +1,5 @@
 # reload SSL certificate around every four hours, in case it was renewed by acme.sh
 0 */4 * * * /home/monitoring/reload-cert.sh > /home/monitoring/last_cert_reload.log 2>&1
+
+# clean up orphan images that are no longer used
+0 0 * * * python ./manage.py cleanupimages
```

Comment:
I don't know if python executable is available by "python"? @fblampe

Reply:
Calling the manage.py implies your inside the checked out repo. I think you need to run this inside the container.

Reply:
Yes, you need to run it inside the app containers. Python is available there.

Reply:
> Calling the manage.py implies your inside the checked out repo. I think you need to run this inside the container. We can have a look together if you like.

That would be great :) When do you have time the next days?

Reply:
Thank you for the input, I have changed it to Github workflow